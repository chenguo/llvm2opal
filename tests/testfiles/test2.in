OPCODE 0xFF2
define void @p1(i32 %a, i32 %a1, i32 %a2, i32 %a3, i32 %a4, i32* %a5, i32* %a6) {
entry:
    %tmp61 = mul i32 %a, %a1        ; <i32> [#uses=1]
    %tmp63 = add i32 %tmp61, %a2        ; <i32> [#uses=1]
    %tmp63 = add i16 %a3, %a4 
}
