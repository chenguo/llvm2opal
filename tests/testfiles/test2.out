DX_RETURNT p1 (DX_PLIST) {
  IREGISTER tmp61 = ( SOURCE1 * S2ORI );
  IREGISTER tmp63 = ( tmp61 + S3ORI );
  tmp63 = ( (int16_t) SOURCE4 + (int16_t) S5ORI );
}

